scripts/globals/pets.lua

```diff
@@ -85,6 +86,9 @@ tpz.pet.id =
 
     --  Puppetmaster
     AUTOMATON             = 69,
+
+    -- Geomancer
+    LUOPAN                = 73,
```

Comment:
```sql

INSERT INTO `pet_list` VALUES (69, 'HarlequinFrame', 5124, 1, 99, 0, 0);
INSERT INTO `pet_list` VALUES (70, 'ValoredgeFrame', 5125, 1, 99, 0, 0);
INSERT INTO `pet_list` VALUES (71, 'SharpshotFrame', 5126, 1, 99, 0, 0);
INSERT INTO `pet_list` VALUES (72, 'StormwakerFrame', 5127, 1, 99, 0, 0);
-- INSERT INTO `pet_list` VALUES (73, 'AdventuringFellow', 0, 1, 99, 0, 0);
-- 74 is Chocobo in the enum..
```
like I've been saying for a long long time now, pet list core code needs changed to not rely on its size and just get the highest ID as the "top" of it so that gaps in the list do not cause issues. 73 was earmarked for teh NPC fellow, but nobody commented it in the lua side of things.

Reply:
I'm going to use 75, i know it was set for trust, but as thats not needed anymore, this shouldnt be a problem, let me know if you feel this should be different.

Reply:
Oh i see why this was 73, its because of the debug break in petutils.cpp, it checks the size of the list, not if the id exsists, so if i set it as 75, it will cause a break, ok let me look into this, i may change this to how i did the trust checking.

Reply:
Ok ive redone LoadPet to better handle gaps in petID's.